src/crypto/crypto_util.cc

```diff
@@ -187,51 +187,86 @@ void TestFipsCrypto(const v8::FunctionCallbackInfo<v8::Value>& args) {
   args.GetReturnValue().Set(enabled);
 }
 
-void CryptoErrorVector::Capture() {
-  clear();
-  while (auto err = ERR_get_error()) {
+void CryptoErrorStore::Capture() {
+  errors_.clear();
+  while (const uint32_t err = ERR_get_error()) {
     char buf[256];
     ERR_error_string_n(err, buf, sizeof(buf));
-    push_back(buf);
+    errors_.emplace_back(buf);
   }
-  std::reverse(begin(), end());
+  std::reverse(std::begin(errors_), std::end(errors_));
 }
 
-MaybeLocal<Value> CryptoErrorVector::ToException(
+bool CryptoErrorStore::Empty() const {
+  return errors_.empty();
+}
+
+template<typename... Args>
+void CryptoErrorStore::Insert(const NodeCryptoError error, Args&&... args) {
```

Comment:
This *might* need to go into a `crypto_util-inl.h` file at some point (because template functions are implicitly also inline functions), but if this works right now, it should be fine :+1:

Reply:
@addaleax I don't think we have a `crypto_util-inl.h`.
I did `tree src | grep "crypto_util\-inl.h"` and couldn't find it. :eyes:

Reply:
Yeah, sorry for being unclear – this would be the first function in that file :)

Reply:
@addaleax Moved it to `src/crypto/crypto_util-inl.h`. I noticed a couple of other template functions and classes in `src/crypto/crypto_util.h`. Should we move those to `src/crypto/crypto_util-inl.h` as well (in another PR)?

Also,
```console
../src/crypto/crypto_util.h:351:14: error: no matching member function for call to 'TrackField'
    tracker->TrackField("errors", errors_);
    ~~~~~~~~~^~~~~~~~~~
```
To fix this, should I again change `CryptoErrorStore` to inherit from `std::vector` or is there another way?